scripts/azure/SharePoint/Reset-SPOFolderAndFilesPermissions.ps1

```
@@ -1,90 +1,480 @@
+            }
+        }
+    }
+
+    # Fallback to loading by name if installed in GAC
+    if (-not $found) {
+        try {
+            Add-Type -AssemblyName "Microsoft.SharePoint.Client, Version=16.0.0.0, Culture=neutral, PublicKeyToken=71e9bce111e9429c" -ErrorAction SilentlyContinue
+            Add-Type -AssemblyName "Microsoft.SharePoint.Client.Runtime, Version=16.0.0.0, Culture=neutral, PublicKeyToken=71e9bce111e9429c" -ErrorAction SilentlyContinue
+            $found = $true
+            return $true
+        } catch { }
```

PSScriptAnalyzer (1.24.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)